examples/coagents-starter-praisonai-agents/agent-py/research/agents.py

```
@@ -0,0 +1,77 @@
+"""PraisonAI Agents for research and reporting"""
+
+from praisonaiagents import Agent, Task, PraisonAIAgents
```

Pylint (3.3.7)

[error] Unable to import 'praisonaiagents'

(E0401)